### PR TITLE
test: cypress - Flaky fix ListV2_nested_List_widget_spec.js

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
Updated Spec: Regression/ClientSide/Widgets/ListV2/ListV2_nested_List_widget_spec.js

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/ListV2_nested_List_widget_spec.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 const commonlocators = require("../../../../../locators/commonlocators.json");
-import { agHelper, propPane } from "../../../../../support/Objects/ObjectsCore";
+import {
+  agHelper,
+  assertHelper,
+  entityExplorer,
+  propPane,
+  draggableWidgets,
+} from "../../../../../support/Objects/ObjectsCore";
+import { AssertHelper } from "../../../../../support/Pages/AssertHelper";
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/ListV2_nested_List_widget_spec.js:9]: Import from ObjectsCore itself. Separate import not required.

Comment (Aishwarya-U-R) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/ListV2_nested_List_widget_spec.js:9]: Remove this unnecessary import to be more precise.

Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/ListV2_nested_List_widget_spec.js]: done

**Review by Aishwarya-U-R — APPROVED** [review]

**Pull request merged**